packages/odata-common/src/request/odata-request.spec.ts

```diff
@@ -11,17 +10,33 @@ import {
 import { commonODataUri } from '../../test/common-request-config';
 import { CommonEntity } from '../../test/common-entity';
 
+// jest.mock('../../http-client/node_modules/axios', () => ({
+//   request: () => Promise.resolve()
+// }));
+//   .spyOn(axios, 'request')
+//   .mockResolvedValue({ 'x-csrf-token': 'test' });
+
+// jest.mock('@sap-cloud-sdk/http-client');
+
+jest.mock('@sap-cloud-sdk/http-client', () => {
+  const actual = jest.requireActual('@sap-cloud-sdk/http-client');
+  return {
+    ...actual,
+    executeHttpRequestWithOrigin: jest
+      .fn()
+      .mockResolvedValue({ 'x-csrf-token': 'test' })
+  };
+});
+
 describe('OData Request', () => {
-  let requestSpy: jest.SpyInstance;
+  // const requestSpy
```

Comment:
remove comments and methods - on my branch I needed no mock at all anymore.

Reply:
Done